src/main/java/com/basho/riak/client/core/util/DefaultCharset.java

```diff
@@ -0,0 +1,99 @@
+package com.basho.riak.client.core.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.charset.Charset;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * <p>
+ *     Holds an application-wide default charset, that
+ *     is then used to encode/decode between Strings and
+ *     Byte arrays for Riak's use (see {@link com.basho.riak.client.core.util.BinaryValue}).
+ *</p>
+ *
+ * <p>
+ *     Before 2.0.3, the system used the JRE's default Charset from the
+ *     {@link Charset#defaultCharset()} property.
+ *</p>
+ *
+ * <p>
+ *     With this class you may change the Riak client to use a different default.
+ *     You can set it at startup by providing the desired Charset name with the vm argument
+ *     {@code -Dcom.basho.riak.client.DefaultCharset="UTF-8"}, or at runtime
+ *     with the static method (see {@link #set(Charset)}).
+ * </p>
+ *
+ * <p>
+ *     As of 2.0.3 it still defaults to the value provided by
+ *     {@link Charset#defaultCharset()},
```

Comment:
It can vary from machine to machine, and from dev to server environments.  We had a customer complain that stuff wasn't getting encoded with UTF-8 by default, which would be the logical choice nowadays.